prices.html

```diff
@@ -303,23 +307,22 @@
         </table>
         <p>&nbsp;</p>
     </div>
-
-    <div class="accs-price-table-wrapper">
-        <table id="table-network-traffic" class="accs-price-table">
-            <thead>
-                <tr>
-                    <th>Data Traffic</th>
-                    <th id="traffic-header-gb-month" style="text-align:right"></th>
-                    <th style="text-align:right">Monthly Free Usage</th>
-                </tr>
-            </thead>
-            <tbody>
-                <tr class="storage-row"><td class="storage">Ingress traffic (RX)</td><td class="gb-month"></td><td class="free">100 GB</td></tr>
-                <tr class="storage-row"><td class="storage">Egress traffic (TX)</td>  <td class="gb-month"></td> <td class="free">100 GB</td></tr>
-            </tbody>
-        </table>
-        <p>&nbsp;</p>
-        </div>
+        <div class="accs-price-table-wrapper">
```

Comment:
I don't really approve of this indent change; why is this div not aligned with the previous?

Reply:
Fixed indentation.